governance-exercises/Le Bateleur - Poll Exercise 01/Course of Action.md

```diff
@@ -0,0 +1,83 @@
+
+Second, the Spork market is distinct from Maker. That ABC token was both onboarded and offboarded from Maker Core due to certain factors (in the indeterminate past) does not necessarily have a direct, one-to-one relevance for Spork. Onboarding a new collateral to Spork adds unique risk, complexity and cost - including, at minimum, the cost of governance and maintaining price feeds. Arguably, an informed decision from Maker voters requires a Spork Protocol-specific risk assessment.
+
+Third, Faraz states that ABC token was offboarded from Maker Core "due to limited usage" and not concerns around liquidity. Yet Maker's offboarding of ABC token due to limited usage arguably throws up concerns, notwithstanding the fact that the Spork market is distinct. A few historical examples expand on my position.
+
+- [As seen here](http://forum.makerdao.com/t/why-does-makerdao-offboard-collateral-that-doesnt-meet-its-net-surplus-requirement/20790/4?u=le_bateleur), Maker Core has offboarded collaterals for low demand/usage since they "are unprofitable after accounting for maintenance and opportunity cost, but also not scalable due to low market liquidity and demand." 
+- Collaterals evincing relatively high on-chain liquidity have nevertheless been offboarded from Maker due to low demand and negative cash flow, as seen in this [instance](https://forum.makerdao.com/t/signal-request-offboard-uni-univ2daieth-univ2wbtceth-univ2unieth-and-univ2wbtcdai/15160). 
+- A collateral with "suboptimal on-chain liquidity" was nevertheless [recommended](http://forum.makerdao.com/t/onboarding-of-gno-to-spark/20831/9?u=le_bateleur) for onboarding to Spark Protocol by Maker's risk assessment team due to other ameliorating factors. 
+
+Thus, an updated, Spork Protocol-specific risk assessment would provide finer resolution for Maker voters into the often multi-faceted question of risk. 
+
+Finally, per the [Poll Principles](https://github.com/makerdao/community/blob/master/processes/poll-principles.md) document, outside observers or "[u]sers of the protocol ... cannot be expected to have the same level of context and knowledge about the inner workings of the Maker protocol as regular voters or delegates." This is another reason why I am concerned about Faraz's Forum reply in which he stated, "*as governance is aware* [emphasis added], ABC was offboarded from Core purely due to limited usage rather than as a result of concern around risk." Without an updated, Spork Protocol-specific risk assessment, I fear we would not be inclusive of these additional stakeholders who cannot be expected to have a comparable command of Maker Core's motivation in offboarding ABC token.  
+
+##### Request Faraz/Advisors confirm their agreement with increasing the Spork Deposit Rate per Alice's proposal.
```

Comment:
Yep. Another good catch. This one was intentional and I think you're the first person to have caught it.